README.md

```diff
@@ -182,7 +182,7 @@ Partition filters can be specified using the syntax described in [HIVE-1609](htt
 
 > The filter supports "=", "!=", ">", "<", ">=", "<=" and "LIKE" operations on partition keys of type string. "AND" and "OR" logical operations are supported in the filter. So for example, for a table having partition keys country and state, the filter can be 'country = "USA" AND (state = "CA" OR state = "AZ")'
 
-In particular notice that it is possible to nest sub-expressions within parentheses.
+In particular notice that it is possible to nest sub-expressions within parentheses. Wild cards are a allowed in the form of `.*`, example: `year like '201.*'`. Full regular expressions doesn't seem to be supported by Hive. 
```

Comment:
"regular expressions doesn't seem..." -> "regular expressions don't seem"

Reply:
Also "are a allowed" -> "are allowed"

Reply:
nice catch yes

Reply:
She's a grammar wizard ! 💯

Reply:
I certainly am not. Thanks for saving my butt!